.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,2 @@
+blank_issues_enabled: true
+version: 0.1
```

Comment:
@dianzrong can you please clarify: what does this file do? What do each of the lines in here do?

Reply:
This file configures the options for new issues. 
The first line allows users to choose to create a blank issue instead of one of the issue templates.
The second line is optional and keeps track of the version number of the Issue Templates.